packages/indexer/src/EventHandlers.ts

```diff
@@ -27,6 +28,13 @@ const RecipientTokenDataParams = [{ name: "depositFee", type: "uint256" }, { nam
 const SplitEtherDataParams = [{ name: "depositFee", type: "uint256" }]
 const SplitTokenDataParams = [{ name: "depositFee", type: "uint256" }, { name: "tokenAddress", type: "address" }]
 
+ShowHubContract_ConditionModuleWhitelisted_loader(({ event, context }) => {
+  const moduleId = `${event.chainId}-${event.params.conditionModule}`
+  context.log.debug(`Processing ShowHubContract_ConditionModuleWhitelisted @ chain ${event.chainId} | Block # ${event.blockNumber}`)
+
+  context.ConditionModule.load(moduleId)
+})
```

Comment:
Since the `ConditionModuleWhitelisted` handler is syncronous, we still need a handler for this to work as expected.